cdk/checkfiles_runner/stacks/runner.py

```diff
@@ -65,6 +69,54 @@ def __init__(
             secret_complete_arn=self.props.portal_secrets_arn
         )
 
+        make_pending_files_checked_message = Pass(
+            self,
+            'MakePendingFilesCheckedMessage',
+            parameters={
+                'detailType': 'PendingFilesChecked',
+                'source': 'RunCheckfilesStepFunction',
+                'detail': {
+                    'metadata': {
+                        'includes_slack_notification': True
+                    },
+                    'data': {
+                        'slack': {
+                            'text': JsonPath.format(
+                                ':white_check_mark: *CheckFilesStarted* | Found {} files in upload_status: pending',
+                                JsonPath.string_at('$.number_of_files_pending')
+                            )
+                        }
+                    }
+                },
+                'files_pending.$': '$.files_pending',
+                'number_of_files_pending.$': '$.number_of_files_pending',
+            },
+        )
+
+        make_checkfiles_finished_message = Pass(
```

Comment:
No explicit failure message?

Reply:
Just the finished message with command status being reported as Failed.